docs/installation-and-operations/installation/docker/README.md

```diff
@@ -70,6 +70,16 @@ your `docker-compose.yml` file is stored (`compose` in this case).
 If you want to start from scratch and remove the exsiting data you will have to remove these volumes via
 `docker volume rm compose_opdata compose_pgdata`.
 
+### Configuration
+
+In the compose folder you will find the file `docker-compose.yml` which can be edited. Please be aware that only those variables shall be edited which are documented as not everything is meant to be configured or bend.
+
+##### BIM Edition
```

Comment:
That title has 5 hashes, while its parent has 3 hashes. Is there something I don't understand or is there one level missing?
```suggestion
#### BIM Edition
```

Reply:
done